client/core/core.go

```diff
@@ -7057,6 +7057,41 @@ func (c *Core) WalletLogFilePath(assetID uint32) (string, error) {
 	return wallet.logFilePath()
 }
 
+// WalletRestorationInfo returns information about how to restore the currently
+// loaded wallet for assetID in various external wallet software. This function
+// will return an error if the currently loaded wallet for assetID does not
+// implement the WalletRestorer interface.
+func (c *Core) WalletRestorationInfo(pw []byte, assetID uint32) ([]*asset.WalletRestoration, error) {
+	crypter, err := c.encryptionKey(pw)
+	if err != nil {
+		return nil, fmt.Errorf("WalletRestorationInfo password error: %w", err)
+	}
+	defer crypter.Close()
+
+	seed, _, err := c.assetSeedAndPass(assetID, crypter)
+	if err != nil {
+		return nil, fmt.Errorf("assetSeedAndPass error: %w", err)
+	}
+	defer encode.ClearBytes(seed)
+
+	wallet, found := c.wallet(assetID)
+	if !found {
+		return nil, fmt.Errorf("no wallet configured for asset %d", assetID)
+	}
+
+	restorer, ok := wallet.Wallet.(asset.WalletRestorer)
+	if !ok {
+		return nil, fmt.Errorf("wallet for asset %d cannot be restored", assetID)
```

Comment:
might wanna be specific and say that "wallet doesn't support exporting functionality"